src/nvim/path.c

```diff
@@ -1098,7 +1102,7 @@ int gen_expand_wildcards(int num_pat, char_u **pat, int *num_file,
     p = pat[i];
 
     if (vim_backtick(p))
-      add_pat = expand_backtick(&ga, p, flags);
+      add_pat = (int)expand_backtick(&ga, p, flags);
```

Comment:
I think is better if I make a direct question: When we do this kind of refactoring is an `assert` **always** necessary when doing a downcast?